src/Bridge/Symfony/Bundle/Resources/public/logo-header.svg

```diff
@@ -1,100 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Be careful @soyuka, you committed the change on the official repository (see https://github.com/api-platform/core/tree/move-swagger) and not on your fork!